relay-server/src/services/store.rs

```diff
@@ -1271,6 +1272,13 @@ impl StoreService {
             );
         }
 
+        trace_item.attributes.insert(
+            "sentry.payload_size_bytes".into(),
+            AnyValue {
+                value: Some(Value::IntValue(span.payload_size_bytes as i64)),
+            },
+        );
```

Comment:
Why do we need it for Spans, the size of a Span is not billing relevant, also in this case, the size used here will completely differ from the one for logs, which makes them not comparable and we will have to change that in the future again (not sure if that is an issue) with more SpanV2 support.

Reply:
It is indeed not billing relevant but we're looking at an indication of the size of the span when we receive it since we'll start applying limits to them soon.

I thought this was the best approximation right now, and we can replace it for a more accurate one later when we have it.